frontend/src/components/dashboard/minesHomePage/Dashboard.js

```diff
@@ -207,38 +229,32 @@ export class Dashboard extends Component {
    */
   handleCoordinateSearch = (value) => {
     if (typeof value === "string") {
+      // Case: searching my mine name
       const newVal = value.split(",");
```

Comment:
This comment looks out of place, either remove or make it more clear

Reply:
removed